sdk/keyvault/keyvault-certificates/review/keyvault-certificates.review.api.md

```diff
@@ -0,0 +1,286 @@
+    keyType?: JsonWebKeyType;
+    // Warning: (ae-forgotten-export) The symbol "KeyUsageType" needs to be exported by the entry point index.d.ts
+    keyUsage?: KeyUsageType[];
+    lifetimeActions?: LifetimeAction[];
+    reuseKey?: boolean;
+    // Warning: (ae-forgotten-export) The symbol "SubjectAlternativeNames" needs to be exported by the entry point index.d.ts
+    subjectAlternativeNames?: SubjectAlternativeNames;
+    subjectName?: string;
+    validityInMonths?: number;
+}
+
+// @public
+export interface CertificateProperties extends ParsedKeyVaultEntityIdentifier {
+    readonly created?: Date;
+    enabled?: boolean;
+    readonly expires?: Date;
+    readonly id?: string;
+    readonly notBefore?: Date;
+    // Warning: (ae-forgotten-export) The symbol "CertificateTags" needs to be exported by the entry point index.d.ts
+    tags?: CertificateTags;
+    readonly updated?: Date;
+    readonly x509Thumbprint?: Uint8Array;
+}
+
+// @public
+export class CertificatesClient {
+    constructor(url: string, credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
+    // Warning: (ae-forgotten-export) The symbol "BackupCertificateResult" needs to be exported by the entry point index.d.ts
+    backupCertificate(name: string, options?: RequestOptionsBase): Promise<BackupCertificateResult>;
+    cancelCertificateOperation(name: string, options?: RequestOptionsBase): Promise<CertificateOperation>;
+    // Warning: (ae-forgotten-export) The symbol "CreateCertificateOptions" needs to be exported by the entry point index.d.ts
+    createCertificate(name: string, certificatePolicy: CertificatePolicy, options?: CreateCertificateOptions): Promise<Certificate>;
+    protected readonly credential: TokenCredential;
+    deleteCertificate(certificateName: string, options?: RequestOptionsBase): Promise<DeletedCertificate>;
+    deleteCertificateContacts(options?: RequestOptionsBase): Promise<Contacts>;
+    deleteCertificateIssuer(issuerName: string, options?: RequestOptionsBase): Promise<CertificateIssuer>;
+    deleteCertificateOperation(name: string, options?: RequestOptionsBase): Promise<CertificateOperation>;
+    getCertificate(name: string, version: string, options?: RequestOptionsBase): Promise<Certificate>;
+    getCertificateContacts(options?: RequestOptionsBase): Promise<Contacts>;
+    getCertificateIssuer(issuerName: string, options?: RequestOptionsBase): Promise<CertificateIssuer>;
+    getCertificateOperation(name: string, options?: RequestOptionsBase): Promise<CertificateOperation>;
+    getCertificatePolicy(name: string, options?: RequestOptionsBase): Promise<CertificatePolicy>;
+    getCertificateWithPolicy(name: string, options?: RequestOptionsBase): Promise<Certificate>;
+    static getDefaultPipeline(credential: TokenCredential, pipelineOptions?: NewPipelineOptions): ServiceClientOptions;
```

Comment:
Is this 'getting' an existing pipeline or creating a new one? I assumed 'getting' because of the name but the parameter type for pipelineOptions is called 'NewPipelineOptions'.